### ISSUE TITLE
Deprecated expressions

### ISSUE THREAD

**Issue by mil1200** [iknowjason/juliart#1]
After running: `terraform apply -var-file=terraform.tfvars -auto-approve`
I have recevied few error messages:

**Warning: Interpolation-only expressions are deprecated**

`  on ../modules/adversary1-vm/1-terraform_azure-create.tf line 6, in resource "azurerm_public_ip" "myterraformpublicip":   6:     location                     = "${var.location}"`

Terraform 0.11 and earlier required all non-constant expressions to be
provided via interpolation syntax, but this pattern is now deprecated. To
silence this warning, remove the "${ sequence from the start and the }"
sequence from the end of this expression, leaving just the inner expression.

Template interpolation syntax is still used to construct strings from
expressions when the template includes multiple interpolation sequences or a
mixture of literal strings and interpolations. This deprecation applies only
to templates that consist entirely of a single interpolation sequence.

(and 122 more similar warnings elsewhere)

**Warning: Quoted references are deprecated**

```
  on ../modules/dc1-vm/4-wait-for-domain-to-provision.tf line 7, in resource "null_resource" "wait-for-domain-to-provision":
   7:   depends_on = ["azurerm_virtual_machine.domain-controller"]
```

In this context, references are expected literally rather than in quotes.
Terraform 0.11 and earlier required quotes, but quoted references are now
deprecated and will be removed in a future version of Terraform. Remove the
quotes surrounding this reference to silence this warning.

(and 19 more similar warnings elsewhere)

**Warning: "address_prefix": [DEPRECATED] Use the `address_prefixes` property instead.**

```
  on ../modules/network/main.tf line 17, in resource "azurerm_subnet" "dc-subnet":
  17: resource "azurerm_subnet" "dc-subnet" {
```

(and 5 more similar warnings elsewhere)

Deployment was succesful and I could connect to the machines, so this is just an informational issue.

**Comment by iknowjason** [comment]
Confirming this issue and thank you.  Will have it fixed as soon as possible.

**Comment by iknowjason** [comment]
Committed update for resolving three fixed issues with deprecated Terraform v0.11 in this repository.
